src/api/app/helpers/branch_package.rb

```diff
@@ -406,7 +409,8 @@ def check_for_update_project(p)
       update_pkg = update_project.find_package(pkg_name, true) # true for check_update_package in older service pack projects
       if update_pkg
         # We have no package in the update project yet, but sources are reachable via project link
-        if update_project.develproject and up = update_project.develproject.find_package(pkg_name)
+        up = update_project.develproject.find_package(pkg_name) if update_project.develproject
+        if update_project.develproject && up
```

Comment:
I don't agree. If ```update_project.develproject``` is false or nil then *up* is not set... so it will fail with a ```NameError: undefined local variable or method 'up' for main:Object``` in the ```if``` statement if I only use it directly without checking first for ```update_project.develproject```